build/configs/rtl8721csm/Make.defs

```diff
@@ -81,6 +81,10 @@ else
   ARCHINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include
   ARCHXXINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../external/include -isystem $(TOPDIR)/../external/include/libcxx
   ARCHXXINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include
+  ARCHXXINCLUDES += -isystem $(TOPDIR)/../framework/include
+ifeq ($(CONFIG_LW_AIFW),y)
+  ARCHXXINCLUDES += -isystem $(TOPDIR)/../framework/src/lw_aifw/include
```

Comment:
We use `framework/include` for public headers. Could you let me know why you need this which locates in `framework/src`?